scripts/install_fonts.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Always add `-e` in order to stop the script on error. Otherwise, it’s like `On Error Resum Next`, if you known what I mean (those good old VBScript days :upside_down_face:).

For an improved debugging experience, you can also add `-x` in order to print each command.

`#!/usr/bin/env bash -e`